components/clinical/patient-banner/src/atoms/ped-number.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC, HTMLAttributes } from 'react'
+import styled from '@emotion/styled'
+import { Group } from '@ltht-react/types'
+
+const StyledPasNumber = styled.div``
+
+const StyledLabel = styled.span`
+  color: #666;
+  font-weight: normal;
+  font-size: 0.75rem;
+`
+
+const StyledValue = styled.span`
+  color: #333333;
+  font-weight: bold;
+  font-size: 0.8125rem;
+  margin-left: 0.5rem;
+`
+
+const PedNumber: FC<Props> = ({ patientGroups, ...rest }) => {
+  const pedGroup = patientGroups.find((pg) => pg.id.includes('PED'))
```

Comment:
`.startsWith` instead?

Reply:
Yeah good shout. It should have been safe anyway as the group identifiers are strictly `"{3-letter-prefix}{6-numeral-id}"` but will change to be extra safe